src/main/java/opendota/Parse.java

```diff
@@ -543,6 +540,20 @@ public void onTickStart(Context ctx, boolean synthetic) {
                             name_to_slot.put(combatLogName2, entry.slot);
 
                             entry.hero_inventory = getHeroInventory(ctx, e);
+                            entry.hero_inventory = getHeroInventory(ctx, e);
+                            if (entry.hero_inventory != null && !isPlayerStartingItemsWritten.get(entry.slot)) {
```

Comment:
The first time hero_inventory is null is after the players have loaded in (after pregame items have been bought?)